### ISSUE TITLE
Corruption/Loss of Data with Excel Data Types (stock, currency, etc)

### ISSUE THREAD

**Issue by dbytes** [dfinke/ImportExcel#1660]
The following will cause corruption or loss of data to an existing sheet (sheet1) that contains Excel Data Types such as stocks.

```
$data = ConvertFrom-Csv @'
Ticker,Date,Units
VOO,11/22/2024,10
VTI,11/20/2024,50
'@

$data | Export-Excel -Path 'invest.xlsx' -WorksheetName 'import' #imports to a new sheet
```

Before:
![before](https://github.com/user-attachments/assets/7346ba79-0444-4993-b57d-013150b45d9b)

After:
![after](https://github.com/user-attachments/assets/7221023a-bf80-42f5-8ddb-69fe01afd7b0)

Excel Workbook as example data:
[invest.xlsx](https://github.com/user-attachments/files/17872822/invest.xlsx)

**Comment by dfinke** [comment]
Does `-append` do what you need?

**Comment by dbytes** [comment]
No, that has the same issue.

**Comment by dfinke** [comment]
if you have plain text in the col A. does it work? not sure of the image and/or the ; colon

**Comment by dbytes** [comment]
If it was plain text it is fine, but this has been converted to an Excel Data Type. 

Setting up the data type, you would enter a ticker such as VOO as plain text in A1 and then convert it.   I've included a GIF to show how to set this up if you were not aware.
![2024-11-22_08-38-11](https://github.com/user-attachments/assets/168101c2-b0d2-459d-af60-f7915160a978)

**Comment by dfinke** [comment]
Yeah, the COM approach is completely different. I don't think the translation in the EPPlus layer is handling it and I don't have control over that.
